governance/xc_admin/packages/xc_admin_frontend/components/tabs/General.tsx

```diff
@@ -0,0 +1,730 @@
+              <tr key={metadataKey}>
+                <td className="base16 py-4 pl-6 pr-2 lg:pl-6">
+                  {metadataKey
+                    .split('_')
+                    .map((word) => capitalizeFirstLetter(word))
+                    .join(' ')}
+                </td>
+
+                <td className="base16 py-4 pl-1 pr-2 lg:pl-6">
+                  {!addNewPriceFeed ? (
+                    <>
+                      <s>{changes.prev[metadataKey]}</s>
+                      <br />{' '}
+                    </>
+                  ) : null}
+                  {changes.new[metadataKey]}
+                </td>
+              </tr>
+            )
+        )}
+      </>
+    )
+  }
+
+  const PriceAccountsChangesRows = ({ changes }: { changes: any }) => {
+    const addNewPriceFeed =
+      changes.prev === undefined && Object.keys(changes.new).length > 0
+    return (
+      <>
+        {changes.new.map((priceAccount: any, index: number) =>
+          Object.keys(priceAccount).map((priceAccountKey) =>
+            priceAccountKey === 'publishers' ? (
+              addNewPriceFeed ? (
+                <PublisherKeysChangesRows
+                  key={priceAccountKey}
+                  changes={{
+                    new: priceAccount[priceAccountKey],
+                  }}
+                />
+              ) : (
+                JSON.stringify(changes.prev[index][priceAccountKey]) !==
+                  JSON.stringify(priceAccount[priceAccountKey]) && (
+                  <PublisherKeysChangesRows
+                    key={priceAccountKey}
+                    changes={{
+                      prev: changes.prev[index][priceAccountKey],
+                      new: priceAccount[priceAccountKey],
+                    }}
+                  />
+                )
+              )
+            ) : (
+              (addNewPriceFeed ||
+                changes.prev[index][priceAccountKey] !==
+                  priceAccount[priceAccountKey]) && (
+                <tr key={priceAccountKey}>
+                  <td className="base16 py-4 pl-6 pr-2 lg:pl-6">
+                    {priceAccountKey
+                      .split('_')
+                      .map((word) => capitalizeFirstLetter(word))
+                      .join(' ')}
+                  </td>
+                  <td className="base16 py-4 pl-1 pr-2 lg:pl-6">
+                    {!addNewPriceFeed ? (
+                      <>
+                        <s>{changes.prev[index][priceAccountKey]}</s>
+                        <br />
+                      </>
+                    ) : null}
+                    {priceAccount[priceAccountKey]}
+                  </td>
+                </tr>
+              )
+            )
+          )
+        )}
+      </>
+    )
+  }
+
+  const PublisherKeysChangesRows = ({ changes }: { changes: any }) => {
+    const addNewPriceFeed = changes.prev === undefined && changes.new.length > 0
```

Comment:
Can you fix this logic so that it doesn't crash when I create a price feed with 0 publishers?